style.css

```diff
@@ -20,32 +20,459 @@ ul {
+  font-weight: 600;
+  font-size: 13px;
+  color: #7a869a;
+}
+
+.work-description {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  color: #344563;
+}
+
+.languages {
+  display: flex;
+  gap: 8px;
+  align-items: center;
+}
+
+.language {
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.card-button {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  padding: 12px;
+  background: #fff;
+}
+
+.card-button:hover {
+  background: #6070ff;
+  color: #fff;
+  outline: none;
+  border: 1px solid #6070ff;
+}
+
+.card-button:focus {
+  background: #2230d2;
+  color: #fff;
+  outline: none;
+  border: 1px solid #2230d2;
+}
+
+.card-button:disabled {
+  border: 1px solid #c1c7d0;
+  color: #5e6c84;
+}
+
+#aboutme {
+  margin-top: 142px;
+  background-color: #fff;
+  display: flex;
+  flex-direction: row;
+  border-radius: 0 100px 0 0;
+  height: 1000px;
+  margin-bottom: -200px;
+  gap: 37px;
+}
+
+.aboutme-button {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  padding: 12px;
+  background: #fff;
+  margin-top: 34px;
+  margin-bottom: 60px;
+}
+
+.aboutme-button:hover {
+  background: #6070ff;
+  color: #fff;
+  outline: none;
+  border: 1px solid #6070ff;
+}
+
+.aboutme-button:focus {
+  background: #2230d2;
+  color: #fff;
+  outline: none;
+  border: 1px solid #2230d2;
+}
+
+.aboutme-button:disabled {
+  border: 1px solid #c1c7d0;
+  color: #5e6c84;
+}
+
+.skill-title {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+}
+
+.skill-list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.skill-detail > p {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+hr {
+  margin-top: 23px;
+  margin-bottom: 24px;
+  background: #dfe1e6;
+}
+
+.textarea:focus {
+  outline: none;
+}
+
+.contact-form-button {
+  background: #fff;
+  border-radius: 8px;
+  padding: 12px 16px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  color: #6070ff;
+  border: none;
+  margin-bottom: 45px;
+}
+
+.contact-form-button:hover {
+  background: #6070ff;
+  color: #fff;
+  outline: none;
+  border: 1px solid #6070ff;
+}
+
+.contact-form-button:focus {
+  background: #2230d2;
+  color: #fff;
+  outline: none;
+  border: 1px solid #2230d2;
+}
+
+.contact-form-button:disabled {
+  border: 1px solid #c1c7d0;
+  color: #5e6c84;
+}
+
+.input:focus {
+  outline: none;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+@media screen and (min-width: 768px) {
```

Comment:
[Optional] Your webpage currently breaks between the widths of `768px` and `1190px`. It would be nice if you added additional breakpoints to make your webpage responsive in these widths🫡

Reply:
Fixed.